### PR TITLE
CI/CD - Improved workflows (cdn/storybook)

### PR DESCRIPTION
**Pull request by MaxencePerrinPrestashop**
- [x] Exported storybook workflow to private repository
- [x] Trigger private actions (cdn and storybook)
- [x] Exported cdn workflow to private repository
- [x] Added documentation

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -26,19 +26,6 @@ jobs:
...
-        run: |
-          echo ::set-output name=TAG::$(echo $GITHUB_REF | cut -d / -f 3)
-          echo ::set-output name=MAJOR::$(echo $GITHUB_REF | cut -d / -f 3 | cut -d '.' -f 1)
-          echo ::set-output name=MINOR::$(echo $GITHUB_REF | cut -d / -f 3 | cut -d '.' -f 2)
-          echo ::set-output name=PATCH::$(echo $GITHUB_REF | cut -d / -f 3 | cut -d '.' -f 3)
-
-      - name: Write .env file with specific CDN URL
-        if: "github.event.release.prerelease"
-        run: echo 'PSX_MKTG_WITH_GOOGLE_CDN_URL=https://storage.googleapis.com/psxmarketing-cdn/${{ steps.get_tag.outputs.MAJOR }}.${{ steps.get_tag.outputs.MINOR }}.${{ steps.get_tag.outputs.PATCH }}/js/' >> .env
-
```
Comment (Quetzacoalt91) [inline .github/workflows/publish.yml:41]: How do we get the version in the next steps?

File: .github/workflows/cdn.yml [hunk 2]
```diff
@@ -11,79 +11,13 @@ jobs:
...
-        if: "!github.event.release.prerelease"
-        run: |
-          gsutil cp -r ./views gs://psxmarketing-cdn/${MAJOR}.x.x
-          gsutil cp -r ./views gs://psxmarketing-cdn/${MAJOR}.${MINOR}.x
-        env:
-          MAJOR: ${{ steps.get_tag.outputs.MAJOR }}
-          MINOR: ${{ steps.get_tag.outputs.MINOR }}
-          PATCH: ${{ steps.get_tag.outputs.PATCH }}
-
+          workflow: metrics-storage.yml
```
Comment (Quetzacoalt91) [inline .github/workflows/cdn.yml:19]: Need to double check this file name.

I wonder if we can have the customizable variable always at the top of our workflow file to make sure we never forget them

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -26,19 +26,6 @@ jobs:
...
       - name: Install composer dependencies
         run: composer install --no-dev -o
-      
-      - name: Get tag
-        id: get_tag
-        run: |
-          echo ::set-output name=TAG::$(echo $GITHUB_REF | cut -d / -f 3)
-          echo ::set-output name=MAJOR::$(echo $GITHUB_REF | cut -d / -f 3 | cut -d '.' -f 1)
-          echo ::set-output name=MINOR::$(echo $GITHUB_REF | cut -d / -f 3 | cut -d '.' -f 2)
-          echo ::set-output name=PATCH::$(echo $GITHUB_REF | cut -d / -f 3 | cut -d '.' -f 3)
```
Comment (Quetzacoalt91) [inline .github/workflows/publish.yml:36]: How do we get the version in the next steps?

File: .github/workflows/cdn.yml (hunk 2 above)
Comment (MaxencePerrinPrestashop) [reply .github/workflows/cdn.yml]: Thanks, wrong workflow here.
Good idea, I created an env on top of our config !

File: .github/workflows/cdn.yml [hunk 4]
```diff
@@ -4,86 +4,23 @@ on:
   release:
     types: [prereleased, released]
 
+env:
+  WORKFLOW: googleshopping-storage.yml
```
Comment (Quetzacoalt91) [inline .github/workflows/cdn.yml:8]: :+1:

File: .github/workflows/cdn.yml [hunk 5]
```diff
@@ -4,86 +4,23 @@ on:
...
-        env:
-          MAJOR: ${{ steps.get_tag.outputs.MAJOR }}
-          MINOR: ${{ steps.get_tag.outputs.MINOR }}
-          PATCH: ${{ steps.get_tag.outputs.PATCH }}
-
+          workflow: ${{ env.WORKFLOW }}
+          repo: PrestaShopCorp/services-deployment
+          token: ${{ secrets.ACCESS_TOKEN }}
+          ref: 'refs/heads/main'
+          inputs: '{ "version": "${{ steps.get_tag.outputs.TAG }}", "nodeVersion": "14", "vuejsPath": "_dev", "vuejsBuildPath": "views", "isRelease": "${{ !github.event.release.prerelease && true || false }}" }'
```
Comment (Quetzacoalt91) [inline .github/workflows/cdn.yml:26]: Same here, the node version should be set as an env var ?

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**